app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -426,6 +450,7 @@ class SuggestedEditsTasksFragment : Fragment() {
     }
 
     companion object {
+        private const val SURVEY_DIALOG_DELAY_TIME_IN_MILLIS = 10000L
```

Comment:
Please add the `CONTRIBUTIONS_DASHBOARD` keyword for this `const`. If you think this is too long, you can consider removing`_TIME_IN_MILLIS`:

```
private const val CONTRIBUTIONS_DASHBOARD_SURVEY_DIALOG_DELAY = 10000L
```

Reply:
I removed this from here and added a default argument for the delay directly within the function. The function has now been moved to ContributionsDashboardHelper.